spec/unit/generator/entry_spec.rb

```diff
@@ -538,8 +538,9 @@ def default_sections
 
         subject { described_class.new(options) }
 
-        it "returns 8 sections" do
-          expect(entry_sections.size).to eq 8
+        it "returns 9 sections" do
+          entry_sections.each { |sec| pp(sec.name) }
```

Comment:
Sorry, line 542 & line 727 are debug code...
`entry_sections.each { |sec| pp(sec.name) }` is unnecessary.